lib/cli/commands/create.ts

```diff
@@ -49,15 +82,30 @@ export const handler = (argv: Arguments) => {
   process.stdout.write('Enter a password: ');
   rl.question('', (password1) => {
     process.stdout.write('\nRe-enter password: ');
-    rl.question('', (password2) => {
+    rl.question('', async (password2) => {
       process.stdout.write('\n\n');
       rl.close();
       if (password1 === password2) {
         const request = new CreateNodeRequest();
         request.setPassword(password1);
-        loadXudInitClient(argv).createNode(request, callback(argv, formatOutput));
+
+        const certPath = argv.tlscertpath ? argv.tlscertpath : getDefaultCertPath();
+        try {
+          await waitForCert(certPath);
+        } catch (err) {
+          console.error(err);
+          process.exitCode = 1;
+          return;
+        }
+
+        const client = loadXudInitClient(argv);
+        // wait up to 3 seconds for rpc server to listen before call in case xud was just started
+        client.waitForReady(Date.now() + 3000, () => {
```

Comment:
Why are waiting 3 seconds for the gRPC server but 5 for the certificate to get created?

Reply:
I just tried to pick numbers I thought were reasonable. I made the maximum wait for the cert slightly longer thinking that the first run of xud might have a slightly longer initialization time, with creating the database and certificates. I'm open to suggestions for change tho.